tests/test_cli/tools_for_testing.py

```diff
@@ -100,3 +105,14 @@ def __init__(self, *args, **kwargs):
     def load(self, *args, **kwargs):
         """Mock the load method."""
         return AEAConfMock()
+
+
+class StopTest(Exception):
+    """An exception to stop test."""
+
+    pass
+
+
+def raise_stoptest(*args, **kwargs):
```

Comment:
@panasevychol where is this being used? I cannot find it

Reply:
https://github.com/fetchai/agents-aea/pull/777/files#diff-1af3034ffb23e6a1564dc6b1b89bab3fR1256